docs/pages/virtual.md

```diff
@@ -11,8 +11,9 @@ layout: default
 | [pipenv]                | Automatically creates and manages a virtualenv for your projects.                                                                                                                                     | 🟠  |
 | [pyenv]                 | Lets you easily switch between multiple versions of Python.                                                                                                                                           | 🟠  |
 | [virtualenv]            | Creates isolated Python environments, and offers more features than venv.                                                                                                                             | 🟠  |
+| [venv]                  | Creates isolated Python environments.                                                                                                                                                                 | 🟠  |
```

Comment:
Maybe we should expand and now justify why it's "equally amber" as `virtualenv`, something like:
```suggestion
| [venv]                  | Creates isolated Python environments. Fewer features than other tools, but very widely used and comes built into Python.                                                                                                                                                     | 🟠  |
```
... or something similar?